src/mbgl/text/glyph.hpp

```diff
@@ -30,6 +30,8 @@ struct GlyphMetrics {
     int32_t left = 0;
     int32_t top = 0;
     uint32_t advance = 0;
+    double ascender = 0.0;
+    double descender = 0.0;
```

Comment:
`double` looks a bit too much from the memory consumption POV, what actual values are set there? Would int32, int16 suffice?

Reply:
Base on FreeType doc, scaled version of ascender and descender are rounded up to an integer value. It makes sense to change them to be int32_t. I will update the proto file in https://github.com/mapbox/node-fontnik/pull/160 accordingly.